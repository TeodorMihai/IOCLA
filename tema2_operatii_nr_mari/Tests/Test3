-2345 + -87D1
