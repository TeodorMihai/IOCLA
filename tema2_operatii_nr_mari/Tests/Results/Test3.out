4523
D187
16AB
-AB16
